lib/tests/cleanup_before.rb

```diff
@@ -25,11 +25,7 @@ def run!(args:)
             # brew doctor complains
             test "sudo", "rm", "-rf", "/usr/local/include/node/"
           elsif OS.mac?
-            if Dir.glob("#{HOMEBREW_CELLAR}/*").present?
-              FileUtils.mkdir_p "/tmp/test-bot-cleanup"
-              # moving is much faster than deleting.
-              test "bash", "-c", "mv #{HOMEBREW_CELLAR}/* /tmp/test-bot-cleanup"
-            end
+            delete_or_move Pathname.glob(HOMEBREW_CELLAR/"*")
```

Comment:
Could remove the `if ENV["HOMEBREW_GITHUB_ACTIONS"] && !ENV["GITHUB_ACTIONS_HOMEBREW_SELF_HOSTED"]` check on line 22 given `delete_or_move` already handles this?

Reply:
No, because the Linux case doesn't use `delete_or_move`.

Reply:
@reitermarkus Might be worth testing if it could?

Reply:
Linux needs `sudo`.